libp2p/Session.cpp

```diff
@@ -39,7 +39,9 @@ Session::Session(Host* _h, unique_ptr<RLPXFrameCoder>&& _io, std::shared_ptr<RLP
 	m_socket(_s),
 	m_peer(_n),
 	m_info(_info),
-	m_ping(chrono::steady_clock::time_point::max())
+	m_ping(chrono::steady_clock::time_point::max()),
+	m_frameWriter(0),
```

Comment:
We will want to have a frame writer for each active protocol (map). Its okay to leave as-is for testing.